### PR TITLE
Create Sitemap Entity Test

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
**Description**
This PR with a test that creates Sitemap

**Steps:**
1. Log in as admin user from data set.
2. Navigate to Marketing > SEO and Search > Site Map.
3. Click "Add Sitemap" button.
4. Fill out all data according to data set.
5. Click "Save" button.
6. Perform all assertions.

**Fixed Issues:**
#338 Convert CreateSitemapEntityTest to MFTF

### CODE REVIEW

File: app/code/Magento/Sitemap/Test/Mftf/ActionGroup/AdminAssertMessageCreateSitemapEntityActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (rogyar) [inline app/code/Magento/Sitemap/Test/Mftf/ActionGroup/AdminAssertMessageCreateSitemapEntityActionGroup.xml]: Please, rename this action group according to the naming convention `Assert<Admin|Storefront><Functionality>ActionGroup. The same suggestion is fair for all other action groups in the scope of the current PR. 

Thank you

File: app/code/Magento/Sitemap/Test/Mftf/Section/SitemapSection.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<sections xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="urn:magento:mftf:Page/etc/SectionObject.xsd">
+    <section name="SitemapGridSection">
+        <element name="add" type="button" selector=".add"/>
```
Comment (rogyar) [inline app/code/Magento/Sitemap/Test/Mftf/Section/SitemapSection.xml]: Please, adjust the simple selectors in the same way as we discussed within the scope of other pull requests. I.e. https://github.com/magento/magento-functional-tests-migration/pull/747#discussion_r351276382

File: dev/tests/functional/tests/app/Magento/Sitemap/Test/TestCase/CreateSitemapEntityTest.xml [hunk 3]
```diff
@@ -13,18 +13,21 @@
...
             <data name="sitemap/data/sitemap_path" xsi:type="string">/</data>
             <constraint name="Magento\Sitemap\Test\Constraint\AssertSitemapFailPathSaveMessage" />
+            <data name="tag" xsi:type="string">mftf_migrated:yes</data>
         </variation>
         <variation name="CreateSitemapEntityTestVariation3">
             <data name="tag" xsi:type="string">severity:S3</data>
             <data name="sitemap/data/sitemap_filename" xsi:type="string">sitemap.xml</data>
             <data name="sitemap/data/sitemap_path" xsi:type="string">/%isolation%</data>
             <constraint name="Magento\Sitemap\Test\Constraint\AssertSitemapFailFolderSaveMessage" />
+            <data name="tag" xsi:type="string">mftf_migrated:yes</data>
```
Comment (rogyar) [inline dev/tests/functional/tests/app/Magento/Sitemap/Test/TestCase/CreateSitemapEntityTest.xml]: Please, do not override an existing `tag` node. Add the `mftf_migrated:yes` as a comma separated value to the existing `tag` node. 

```xml
<data name="tag" xsi:type="string">severity:S3,mftf_migrated:yes</data>
```

Comment (rogyar) [reply dev/tests/functional/tests/app/Magento/Sitemap/Test/TestCase/CreateSitemapEntityTest.xml]: The same suggestion is fair for all newly added `tag` nodes.

File: app/code/Magento/Sitemap/Test/Mftf/Section/SitemapSection.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        <element name="messageByType" type="block" selector="#messages .message-{{messageType}}" parameterized="true"/>
+        <element name="search" type="button" selector=".admin__filter-actions button[title=Search]"/>
+        <element name="resetFilter" type="button" selector=".action-tertiary[title='Reset Filter']"/>
+        <element name="searchResultFirstRow" type="text" selector=".data-grid>tbody>tr"/>
+        <element name="name" type="input" selector="#sitemapGrid_filter_sitemap_filename"/>
+        <element name="rowByIndex" type="text" selector="tr[data-role='row']:nth-of-type({{var1}})" parameterized="true" timeout="30"/>
+
+    </section>
+    <section name="AddSiteMapSection">
```
Comment (rogyar) [inline app/code/Magento/Sitemap/Test/Mftf/Section/SitemapSection.xml]: Please, use one section per file. If we need two sections, put every section to a separate file.

**Comment by DmitryTsymbal** [comment]
Hello @rogyar 
Thank you for the review.
I have made needed changes in the new PR: https://github.com/magento/magento-functional-tests-migration/pull/759
Could you please check it?
Thanks!

**Pull request closed without merging**